public/examples/js/plugin-filters/outline.js

```diff
@@ -3,7 +3,7 @@ document.body.appendChild(app.view);
 
 app.stage.position.set(400, 300);
 
-const outlineFilterBlue = new PIXI.filters.OutlineFilter(2, 0x99ff99);
+const outlineFilterBlue = new PIXI.filters.OutlineFilter(2, 0x9999ff);
```

Comment:
Can you change the variable name to `outlineFilterGreen` instead?

Reply:
Hi, I just read your reply. I've changed the variable name instead, just like you said.